learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md

```diff
@@ -80,27 +81,21 @@ Follow the steps below to write a simple program in a Ballerina package, which p
     }
     ```
 
-    This function prints “Hello World!” in the console. 
+    This function prints `Hello World!` in the console. 
+
+3. Navigate to the created `helloworld` package directory.
 
-3. Execute the `bal build` command to build an executable of this file. You view the output below.
+4. Execute the `bal build` command to build an executable of this file. You view the output below.
 
     ```bash
     Compiling source
-        examples/helloworld:0.1.0
-    
-    Running Tests
-    
-        helloworld
-        No tests found
-    
-    Creating the BALA file
-        target/bala/examples-helloworld-any-0.1.0.bala
-    
+	    praneesha/test_new:0.1.0
+
     Generating executable
-        target/bin/helloworld.jar
+	    target/bin/test_new.jar
```

Comment:
This should not be changed. It's `target/bin/helloworld.jar` in this case since the package name is `helloworld`

Reply:
Fixed.